### PR TITLE
feat: cluster mode in Neo Session launcher 

### PR DESCRIPTION
**Pull request by yomybaby**
ref [Task list ](https://github.com/lablup/backend.ai-webui/issues/2025)

- Cluster form items
  image

- Cluster & total allocations
  image

Please test cluster mode using Backend.AI server supports cluster mode such as `xxx.xx.64.15` test server(with cluster size 1 and 2).

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: react/src/components/ResourceAllocationFormItems.tsx [hunk 1]
```diff
@@ -391,7 +379,7 @@ const ResourceAllocationFormItems: React.FC<
         if (nextImageSelectorType) {
```
Comment (lizable) [inline react/src/components/ResourceAllocationFormItems.tsx:379]: I think we should add else block with setting values in accelerator value to zero(`0`) considering the image doesn't support AI accelerator. If not, the last page of the session launcher would display AI accelerator value, but when after creating session, the accelerator does not allocated to session, which may confuse user.

Here's the session that I tried to create using simple python kernel.

### Screenshot(s)
- The last page of session launcher (1 Warboy included)
Screenshot 2023-11-15 at 8 15 17 PM

- just created session from session launcher in session list
Screenshot 2023-11-15 at 8 15 44 PM

File: react/src/components/ResourceAllocationFormItems.tsx [hunk 2]
```diff
@@ -776,77 +747,125 @@ const ResourceAllocationFormItems: React.FC<
...
-                            min: sliderMinMax[currentAcceleratorType]?.min || 0,
+                            min:
+                              sliderMinMaxLimit[currentAcceleratorType]?.min ||
+                              0,
                           },
                           remainingValidationRules[currentAcceleratorType],
                         ]}
-                      />
+                      >
+                        <InputNumberWithSlider
```
Comment (lizable) [inline react/src/components/ResourceAllocationFormItems.tsx:757]: And I think we need to disable the AI accelerator input field when image does not support AI accelerator (e.g. simple python kernel).

**Review by lizable — CHANGES_REQUESTED** [review]
I checked multi-node cluster session creation and It works fine, but there are a few things need to be fixed at ResourceAllocationFormItems component, especially in AI accelerator input field. I left comments below.

**Comment by lizable** [comment]
> I checked multi-node cluster session creation and It works fine, but there are a few things need to be fixed at ResourceAllocationFormItems component, especially in AI accelerator input field. I left comments below.

This will be handled as a separate issue.

**Review by lizable — APPROVED** [review]
LGTM.

**Comment by yomybaby** [comment]
Thank you for reviewing. 💪 
The accelerator value and disabling issue that you mentioned have been added https://github.com/orgs/lablup/projects/24/views/16?filterQuery=-last-updated%3A7days+beta&pane=issue&itemId=44660836

**Pull request merged**